src/snowflake/connector/aio/auth/_default.py

```diff
@@ -4,13 +4,20 @@
 
 from __future__ import annotations
 
+from logging import getLogger
 from typing import Any
 
 from ...auth.default import AuthByDefault as AuthByDefaultSync
 from ._by_plugin import AuthByPlugin
 
+logger = getLogger(__name__)
+
 
 class AuthByDefault(AuthByPlugin, AuthByDefaultSync):
```

Comment:
lets name `AuthByPlugin` as `AuthByPluginAsync` for clarity?

Reply:
will change it